src/main/java/com/google/devtools/build/lib/remote/RemoteRetrier.java

```diff
@@ -154,6 +154,7 @@ public static class ExponentialBackoff implements Backoff {
      */
     ExponentialBackoff(Duration initial, Duration max, double multiplier, double jitter,
         int maxAttempts) {
+      Preconditions.checkArgument(max.compareTo(initial) > 0, "max must be > initial");
```

Comment:
As a rule, Bazel should not crash due to malformed input. To avoid additional validation logic, how about we use `max(initial, remoteRetryMaxDelay)` as the argument to `ExponentialBackoff` below?